tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1451,22 +1450,21 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		time.Sleep(time.Duration(pandoraSyncWaitTime) * time.Second)
 
 		ginkgo.By("Create CNS register volume with above created FCD")
-		log.Infof("Auto created namespace :%s", autocreatednamespace)
+		framework.Logf("Auto created namespace :%s", autocreatednamespace)
 		cnsRegisterVolume := getCNSRegisterVolumeSpec(ctx, autocreatednamespace, fcdID, "", pvcName, v1.ReadWriteOnce)
 		err = createCNSRegisterVolume(ctx, restConfig, cnsRegisterVolume)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		time.Sleep(time.Duration(40) * time.Second)
+		framework.ExpectError(waitForCNSRegisterVolumeToGetCreated(ctx, restConfig, namespace, cnsRegisterVolume, poll, pollTimeoutShort))
 		cnsRegisterVolumeName := cnsRegisterVolume.GetName()
-		log.Infof(" CNS register volume name : %s", cnsRegisterVolumeName)
+		framework.Logf(" CNS register volume name : %s", cnsRegisterVolumeName)
 
 		ginkgo.By("Verify the error message, when FCD without storage policy is used")
 		cnsRegisterVolume = getCNSRegistervolume(ctx, restConfig, cnsRegisterVolume)
 		actualErrorMsg := cnsRegisterVolume.Status.Error
-		log.Infof("Error message :%s", actualErrorMsg)
+		framework.Logf("Error message :%s", actualErrorMsg)
 		expectedErrorMsg := fmt.Sprintf("Failed to find K8S Storageclass mapping storagepolicyId: %s and assigned to namespace: %s", profileID, autocreatednamespace)
+		framework.Logf("Error message :%s", expectedErrorMsg)
 		if !strings.Contains(actualErrorMsg, expectedErrorMsg) {
-			log.Errorf("Expected error message : %s", expectedErrorMsg)
-			log.Errorf("Actual error message : %s", actualErrorMsg)
 			gomega.Expect(actualErrorMsg).NotTo(gomega.HaveOccurred())
 		}
```

Comment:
Change to : gomega.Expect(strings.Contains(actualErrorMsg, expectedErrorMsg), gomega.BeTrue())

Reply:
done